packages/react-dom/src/events/DOMEventResponderSystem.js

```diff
@@ -436,6 +441,7 @@ function processTimers(
 
 function createDOMResponderEvent(
   topLevelType: string,
+  responderTarget: Element,
```

Comment:
Any event going to a responder should always have a `responderTarget` set, and it does, but this Flow type means all the responders need to account for a `null` case.